### PR TITLE
[TE] Self-Serve Onboarding 04: enhancements

### PR DESCRIPTION
**Pull request by newsummit**
Takes care of:
- Loading severity score from backend
- Simplifying transition error states (like from create alert >> alert page)
- New alerts to be saved with an "owner"
- Resolving presentational issues
- Removing template/controller code related to obsolete logic (for example, we're not waiting for replay status any more)

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js [hunk 1]
```diff
@@ -69,6 +69,26 @@ const fetchCombinedAnomalies = (anomalyIds) => {
...
 };
 
+/**
+ * Fetches severity scores for all anomalies
+ * TODO: Move this and other shared requests to a common service
+ * @param {Array} anomalyIds - list of all found anomaly ids
+ * @returns {RSVP promise}
+ */
+const fetchSeverityScores = (anomalyIds) => {
+  if (anomalyIds.length) {
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js]: Do we need a check against anomalyIds being `undefined`/`null` here?

File: thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs [hunk 2]
```diff
@@ -285,19 +287,13 @@
     </div>
 
   {{else}}
-
     <div class="te-alert-page-pending">
-      <img src="{{rootURL}}assets/images/te-alert-{{if isReplayStatusError "error" "pending"}}.png" class="te-alert-page-pending__image {{if isReplayStatusError "te-alert-page-pending__image--error"}}" alt="alertData.Setup is Processing">
-      <h2 class="te-alert-page-pending__title">{{if isReplayStatusError "Oops, something went wrong." "Setting up your alert"}}</h2>
-      {{#if (not isReplayStatusError)}}<div class="te-alert-page-pending__loader"></div>{{/if}}
+      <img src="{{rootURL}}assets/images/te-alert-pending.png" class="te-alert-page-pending__image" alt="alert setup processing">
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs]: Did {{rootURL}} work?

File: thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js (hunk 1 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js]: yes. thanks.

File: thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs]: I believe it was working all along, I just had some complex interfering boolean conditionals blocking it.

**Pull request merged**